nss-day-cohort-47/tabloid-mvc-reese-s-puffs
# Add Comment

# Description

When a user clicks on the create comment link on the comment view page, they should be taken to a new page where they are able to input subject and content fields. If the user decides to not leave a comment, they can hit the back to list button to return to the post comment page. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

go to Post, view comments, create comment. A saved comment should return the user to the comment list page where their comment should be displayed. 



# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
